ecs-cli/modules/commands/compose/command_test.go

```diff
@@ -1,49 +0,0 @@
-// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License"). You may
-// not use this file except in compliance with the License. A copy of the
-// License is located at
-//
-//	http://aws.amazon.com/apache2.0/
-//
-// or in the "license" file accompanying this file. This file is distributed
-// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-// express or implied. See the License for the specific language governing
-// permissions and limitations under the License.
-
-package composeCommand
```

Comment:
Nope. This test is moved to https://github.com/aws/amazon-ecs-cli/pull/241/files/967f1a7d31c9b358f05f24309421b874f7571b8c..fe57e54fd49f113407fb01a700d482ed34fc7cc4#diff-dbff8407fbb34f9449a0df81b62f61d1R71 also due to the import cycle restriction.